session.go

```diff
@@ -122,30 +123,16 @@ func (s *Session) Passive(nw string) error {
 		s.Data.Close()
 		s.Data = nil
 	}
-	li, err := s.Server.listen(nw, s.passiveHost()+":0")
+	addr := net.JoinHostPort(s.Server.Host, "0")
```

Comment:
This seems backwards. I'd expect you to always be binding to the IP of the control channel, but sending `s.Server.Host` as the response to the PASV command only if it is set (which implies NAT).